templates/bootstrap/plugin_name/app/viewsets.py.j2

```diff
@@ -11,7 +11,7 @@ from rest_framework import status
 from rest_framework.decorators import action
 from rest_framework.response import Response
 
-from pulpcore.app.viewsets import RemoteFilter
+from pulpcore.plugin.viewsets import RemoteFilter
```

Comment:
:smile:

Reply:
Looks like it's missing from the plugin API. Fix: https://github.com/pulp/pulpcore/pull/666